bedrock/firefox/redirects.py

```diff
@@ -600,4 +600,8 @@ def firefox_channel(*args, **kwargs):
 
     # issue 6512
     redirect(r'^firefox/firefox\.html$', 'firefox.new'),
+
+    # bug 1511420
+    redirect(r'^firefox/tracking-protection/start/?$', '/firefox/content-blocking/start/'),
+    redirect(r'^firefox/(?P<version>\d+\.\d+(?:\.\d+)?(?:a\d+)?)/tracking-protection/start/?$', '/firefox/{version}/content-blocking/start/'),
```

Comment:
@pmac this PR isn't ready to merge, but thought I'd ask you for a review here, mainly for this redirect logic. The Firefox team has asked to change the URL for this view from `/tracking-protection/` to `/content-blocking/`. I added some tests and they seem to be passing, but a second pair of eyes here would be good.